packages/medusa/src/commands/start.ts

```diff
@@ -24,7 +25,7 @@ const CRON_SCHEDULE = EVERY_SIXTH_HOUR
  */
 async function registerInstrumentation(directory: string) {
   try {
-    const instrumentation = await import(
+    const instrumentation = await dynamicImport(
```

Comment:
suggestion: can we add a test so this doesn't happen again?

Reply:
I ll add on the second pr ill make

Reply:
Cause it will make more sense than testing the import module from nodejs 😊 the next one will be more granular

Reply:
Yes, don't think that is our responsibility 😄